resolwe_docker_images/base/Dockerfile.ubuntu-22.04

```diff
@@ -1,4 +1,6 @@
-FROM public.ecr.aws/genialis/resolwe/base:ubuntu-22.04-24102023
+ARG BASE_IMAGE=public.ecr.aws/genialis/resolwe/base:ubuntu-22.04-24102023
```

Comment:
Perhaps set the default argument to some generic tag, like `latest` instead. Otherwise these tags are going to get replaced whenever the BASE_IMAGE is omitted in `build.env`, and update some old version of the tag.

Reply:
OK.